plutus-ledger-api/test/Spec/CostModelParams.hs

```diff
@@ -29,11 +27,8 @@ tests =
     "costModelParams"
     [ testCase "length" $ do
             166 @=? length v1_ParamNames
-            166 @=? length V1.costModelParamsForTesting
```

Comment:
Is this no longer relevant? If so, why?

Reply:
Those tests don't really make sense any more.  This is the stuff that needs to be reworked in #6089.